src/thread_rng.rs

```diff
@@ -146,7 +143,7 @@ impl CryptoRng for ThreadRng {}
 /// If you're calling `random()` in a loop, caching the generator as in the
 /// following example can increase performance.
 ///
-/// ```
+/// ```rust
```

Comment:
I think the `rust` is redundant. (However, I guess it does not hurt to be explicit.)